dotcom-rendering/src/components/marketing/epics/ContributionsLiveblogEpic.tsx

```diff
@@ -208,6 +208,12 @@ export const ContributionsLiveblogEpic: ReactComponent<EpicProps> = ({
 		return <></>;
 	}
 
+	const variantOfChoiceCard =
```

Comment:
should the liveblog epic also have the `US_THREE_TIER_CHOICE_CARDS` variant?

Reply:
ahh good question! I assumed we would want to change on the live blog epic as it might look at bit weird to have one epic showing one set of choice cards and then another showing different options.

Reply:
I'm not sure...could check with Jenny on Monday?

Reply:
Yes, there should be the US variant for the Live Blogs. I've added it as part of this change!